src/audio/google/google_ctc_audio_processing.h

```diff
@@ -37,10 +37,23 @@ struct google_ctc_audio_processing_comp_data {
 	uint32_t chunk_frames;
 	GoogleCtcAudioProcessingState *state;
 	struct comp_data_blob_handler *tuning_handler;
+	struct google_ctc_config *config;
 	bool reconfigure;
 	ctc_func ctc_func;
 };
 
+struct google_ctc_config {
+	uint32_t size;
```

Comment:
please define if this is inclusive or just the size of `data`

Reply:
done. Thanks!